packages/client-core/src/admin/components/avatar/AvatarTable.tsx

```diff
@@ -129,12 +129,18 @@ export default function AvatarTable({ search }: { search: string }) {
                 />
               )
             }}
-          >
-            <HiTrash className="place-self-center text-theme-iconRed" />
-          </Button>
+            startIcon={<HiTrash className="place-self-center text-theme-iconRed" />}
+          />
         </div>
       )
     }))
 
-  return <DataTable query={adminAvatarQuery} columns={avatarColumns} rows={createRows(adminAvatarQuery.data)} />
+  return (
+    <DataTable
+      className="h-[calc(100vh_-_318px)]"
```

Comment:
If we have to add this property on all tables, can we make it a behavior of `DataTable` control by default? Or if required we can make it toggle able through a property `isFullScreen` or `isResponsive`?

Reply:
Yes, by default the table adjusts its height to fit the number of rows it can render (up to a limit). However, the desired behavior is for the tables to always fit the screen size. The way I made that possible was by calculating the height of each table page because each page has a different layout, such as header filters, tabs, or page tools in the top. This ensures there are no unexpected vertical scrolls or hidden content.

One possible solution could be to introduce 4 different sizes, for example: `sm`, `md`, `lg`, and `xl`. However, as you can see, most table pages don't match exactly across sizes, and this could lead to issues like unexpected vertical scrolls, hidden content, etc.

Reply:
i think the options with the variants by size is cleaner just implemented, what do you think ?